### PR TITLE
Allow all origins, document endpoints, remove static serving

### PR DESCRIPTION
**Pull request by tomdyson**

### CODE REVIEW

**Comment by tomdyson** [comment]
@zerolab I'm going to go ahead and merge this, hopefully uncontroversial enough! But we might want to support env-specified origins in the future.

File: nlbq/api.py [hunk 1]
```diff
@@ -12,6 +12,17 @@
 
 app = FastAPI()
 
+# Allow all origins
+origins = ["*"]
```
Comment (zerolab) [inline nlbq/api.py:16]: suggestion: make this configurable

```python
# config.py

class Settings(BaseSettings):
    ...
    allow_origins: list[str] = ["*"]
```

then here `allow_origins=settings.allow_origins`

Comment (zerolab) [reply nlbq/api.py:16]: note to self: https://fastapi.tiangolo.com/tutorial/cors/

**Review by zerolab — APPROVED** [review]
I'm not fast enough. made a small suggestion which I can take care of in a separate PR

File: nlbq/api.py (hunk 1 above)
Comment (tomdyson) [reply nlbq/api.py:16]: Your suggestion is perfect, thanks!

Comment (zerolab) [reply nlbq/api.py:16]: #18

**Pull request merged**